llama_stack/apis/post_training/post_training.py

```diff
@@ -201,11 +176,8 @@ async def preference_optimize(
     @webmethod(route="/post-training/jobs", method="GET")
     async def get_training_jobs(self) -> ListPostTrainingJobsResponse: ...
 
-    @webmethod(route="/post-training/job/status", method="GET")
-    async def get_training_job_status(self, job_uuid: str) -> Optional[PostTrainingJobStatusResponse]: ...
-
-    @webmethod(route="/post-training/job/cancel", method="POST")
```

Comment:
Cancel should be supported. There should also be - separately - a remove method (that can be backed by DELETE).